ihsanbakidogan/dist/presence.js

```diff
@@ -0,0 +1,45 @@
+var presence = new Presence({
+    clientId: "703601006994260131"
+  });
+  
+var browsingStamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+  let presenceData = {
+    largeImageKey: "ibd",
+    startTimestamp: browsingStamp
+  };
+  if (document.location.hostname == "ihsanbakidogan.com") {
+	  
+    if (document.location.pathname.startsWith("/index")) {
+      presenceData.details = "şu anda Ana Sayfaya,";
+      presenceData.state = "göz atıyor...";
+	    
+    } else if (document.location.pathname.startsWith("/hakkimda")) {
+      presenceData.details = "şu anda Hakkımda sayfasına,";
+      presenceData.state = "göz atıyor...";
+	   
+	      	      	      
+      } else if (document.location.pathname.includes("/blog/gonderi/")) {
+      presenceData.details = document.title;
+      presenceData.state = "adlı yazıyı okuyor...";
+	      	
+
+      } else if (document.location.pathname.includes("/kategori/")) {
+      presenceData.details = document.title + " adlı";
+      presenceData.state = "kategorideki yazılara bakıyor...";
+	     			
+	      	      	      
+	      							   
+
+
+
+
+

```

Comment:
remove unneeded enters, TIP: Use prettier extension if you use vscode

Reply:
or Atom